.github/workflows/codecov.yml

```diff
@@ -40,3 +42,5 @@ jobs:
       uses: codecov/codecov-action@v5
       with:
         files: coverage.xml
+    permissions:
+      actions: read
```

Comment:
Based on https://github.com/ioos/compliance-checker/issues/1148, we may need higher permissions here. Not sure.